CMSgov/easi-app
# Task List Section Status Logic - Fill out intake form request(1) FE

# EASI-3059

[Task list figma](https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=475-110638&mode=design&t=lkdaDgyD1dmXyrID-0)
[Task statuses figma](https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=12-23756&mode=design&t=lkdaDgyD1dmXyrID-0)

## Changes and Description

- New component folder `GovTaskIntakeForm`
- Property and layout updates to `TaskListItem` + `TaskStatusTag`



## Testing
- Jest tests for `GovTaskIntakeForm`
- Preview on local dev
  - Check out the preview branch `EASI-3059/gov-task-list-1-mock-preview` which pulls in mock data for all status states within the parent task list component
  - Turn on the `itGovV2Enabled` flag
  - Seed data or create a new IT Gov request to get to the task list view
